Changelog.rst

```diff
@@ -613,7 +656,12 @@ pupmod-simp-useradd
 simp-core
 ^^^^^^^^^
 
-* Add logic to auto.cfg to use OS-specific GPG keys in simp_filesystem.repo
+* Add logic to auto.cfg to use OS-specific GPG keys in simp_filesystem.repo.
+
+simp-doc
+^^^^^^^^
+
+* Updated the documentation for UEFI PXE booting.
```

Comment:
Missing:

- Added HOWTO to IPA enrollment workflow
- Added documentation on how to disconnect from puppetDB if needed
- Updated contributors guide to describe additional details about development workflow
- Added FAQ clarifying different scenarios for managing SSH
- Removed OBE  MCollective references
- Added SIMP on AWS documentation